syntax = "proto2";

package RoomInfo;

message RoomInformation {
    required string host = 1;
    required string map = 2;
    required int32 capacity = 3;
    required int32 size = 4;
    required bool active = 5;
    repeated int32 colors = 6;
    repeated string players = 7;
    repeated int32 types = 8;
    repeated bool ready = 9;

}

message RoomInfoPackage {
    repeated RoomInformation roominfo = 1;
}
